### DISCUSSION TITLE
how to extract critical classes to another CSS file?

### DISCUSSION THREAD

**Discussion by nekooee** [tailwindlabs/tailwindcss#5728] [Help] [answered]
Hi,
I need to extract none-critical classes to another CSS file and use it this by `rel="preload" as="style"`

How can I do this?

![image](https://user-images.githubusercontent.com/6003803/136370249-74821666-b356-4c94-94be-852f488f46b4.png)

**Comment by simonswiss** [comment, marked as answer]
Hey!

This is not something Tailwind CSS will do, but you can use a tool like [Critical](https://github.com/addyosmani/critical) to achieve this, as a "build" step (after your CSS is generated by Tailwind).
